### PR TITLE
Import cce

### LINKED ISSUE #121: Importing Cold chain assets: add to the docs what happens in case import fails/encounters fails
Currently, the docs only talk about what happens during a successful import of Cold chain assets. However, it would be extremely useful to know what to do when the import fails/encounters errors along the way. 

For example, it is not clear that the user can't see all errors at the same time, and will only see errors of duplicated assets in the import file -> and THEN in omSupply. E.g. it is not clear the errors are nested and don't show at the same time.

Raising after the discussion related to this issue: https://github.com/msupply-foundation/open-msupply/issues/5467

### LINKED ISSUE #122: Add options for statuses when importing CCE
Can we add these to the docs somewhere, so people know what options they have for that column?

_Originally posted by @jmbrunskill in https://github.com/msupply-foundation/open-msupply/pull/5553#discussion_r1857341303_

Also the date formats have changed so we need to update to reflect this...

### PR DESCRIPTION
**Pull request by lache-melvin**
Fixes #122 
Fixes #121 

Adds:
- Explanation of errors/warnings on CCE import
- Functional status options
- Link to where the asset catalogue codes come from
- Fixes asset delete screenshot

### CODE REVIEW

File: content/docs/coldchain/equipment.md [hunk 1]
```diff
@@ -187,6 +221,15 @@ The `Details` tab shows the asset properties.
...
 The `Status History` tab shows the history of the functional status of the asset.
 
+Available functional statuses are:
+
+- `Functioning`
+- `Functioning but needs attention`
+- `Not functioning`
+- `Not in use`
+- `Unserviceable`
+- `Decommissioned`
```
Comment (roxy-dao) [inline content/docs/coldchain/equipment.md:231]: I think these can just be a list and doesn't need to be highlighted as well

**Review by roxy-dao — CHANGES_REQUESTED** [review]
Whoooo

File: content/docs/coldchain/equipment.md [hunk 2]
```diff
@@ -82,34 +82,68 @@ template which can be used to import assets.
...
+| Column                  | Description                                                                                | Example                    | Mandatory         |
+| :---------------------- | :----------------------------------------------------------------------------------------- | :------------------------- | :---------------- |
+| **Store**               | The code of the Store to which the asset is to be assigned                                 | CMS                        | \* (central only) |
+| **Asset Number**        | The asset number assigned by the user's organisation                                       | 1                          | \*                |
+| **Catalogue item code** | The [Asset Catalogue code](/docs/catalogue/assets) - i.e. the kind of asset to be imported | E003/002                   | \*                |
+| **Installation date**   | The date that the asset was installed at the Store                                         | 2024-01-14                 |                   |
+| **Replacement due**     | The date that the asset is due for replacement                                             | 2026-01-14                 |                   |
```
Comment (jmbrunskill) [inline content/docs/coldchain/equipment.md:97]: Looks like this is now DD/MM/YYYY? After Noels changes.

Also it's not consistent in the UI anymore :(

image


Personally I'm not convinced on the change to DD//MM/YYYY for imports, I like the old format better...

File: content/docs/coldchain/equipment.md [hunk 3]
```diff
@@ -82,34 +82,68 @@ template which can be used to import assets.
...
+
+After uploading your CSV file, your data will be validated and displayed to you to review. At this step, if the data in your uploaded file is not valid, you will see an error message and won't be able to continue to the import step.
+
+![](/docs/coldchain/images/cce_import_error.png)
+
+Common mistakes here are uploading a file type other than CSV, or if you included some duplicate asset numbers in your file. You'll see these errors in the `Error message` column.
+
+You can use the `Export` button at the bottom of the upload window to download a CSV file which includes the error messages, to investigate what went wrong.
+
+Once you have corrected the errors, you can go back to the `Upload` tab to upload the corrected file.
```
Comment (jmbrunskill) [inline content/docs/coldchain/equipment.md:126]: I think it's almost worth having a note about date formats, as that's the one that generally gets automatically mangled by spreadsheets?

**Review by jmbrunskill — APPROVED** [review]
Looks good, only thing might be the date formats stuff, but feel like there might be a bit buggy behaviour there anyway...

File: content/docs/coldchain/equipment.md (hunk 2 above)
Comment (lache-melvin) [reply content/docs/coldchain/equipment.md:97]: Ugh yeah, we made about 4 PRs into this modal at the same time, I think it needs one more. I don't really mind the date format either way, but if one maps better to/from the exports, I'd be inclined to go with that. 

Probably not high priority now 😢 Will update here to be consistent, thanks for catching 😅

File: content/docs/coldchain/equipment.md (hunk 3 above)
Comment (lache-melvin) [reply content/docs/coldchain/equipment.md:126]: Yeah good idea! Will add :pray:

**Pull request merged**